examples/tinywl-new/output.cpp

```diff
@@ -86,14 +87,7 @@ void Output::addSurface(SurfaceWrapper *surface)
 
     if (surface->type() == SurfaceWrapper::Type::Layer) {
         auto layer = qobject_cast<WLayerSurface*>(surface->shellSurface());
-        layer->safeConnect(&WLayerSurface::ancherChanged, this, &Output::layoutLayerSurfaces);
-        layer->safeConnect(&WLayerSurface::leftMarginChanged, this, &Output::layoutLayerSurfaces);
-        layer->safeConnect(&WLayerSurface::rightMarginChanged, this, &Output::layoutLayerSurfaces);
-        layer->safeConnect(&WLayerSurface::topMarginChanged, this, &Output::layoutLayerSurfaces);
-        layer->safeConnect(&WLayerSurface::bottomMarginChanged, this, &Output::layoutLayerSurfaces);
-        layer->safeConnect(&WLayerSurface::exclusiveZoneChanged, this, &Output::layoutLayerSurfaces);
-        connect(surface, &SurfaceWrapper::widthChanged, this, &Output::layoutLayerSurfaces);
```

Comment:
Should use `desiredSize` for SurfaceWrapper?

Reply:
DesiredSizeChanged is also part of layerPropertiesChanged